File: 단계별로 풀어보기/Step 06/4344.cpp
#include <iostream>
using namespace std;

int main() {
    int C;
    cin >> C;

    int i;
    for(i=0; i<C; i++) {
        int N;
        cin >> N;
        int *score = new int[N];
        int j;
        int sum = 0;
        float avg;
        for(j=0; j<N; j++) {
            cin >> score[j];
            sum += score[j];
        }
        avg = (float)sum/N;
        int cnt = 0;
        for(j=0; j<N; j++) {
            if (score[j] > avg) cnt++;
        }
        cout << fixed;
        cout.precision(3);
        cout << (float)cnt/N*100 << "%" << endl;
    }
    return 0;
}

File: StepByStep/Python/Step_04_1차원 배열/4344.py
C = int(input())
for i in range(C):
    nums = list(map(int, input().split()))
    avg = sum(nums[1:]) / nums[0]
    cnt = 0
    for score in nums[1:]:
        if avg < score:
            cnt += 1
    print(str("{:.3f}".format(cnt/nums[0]*100)) + '%')